articles/stream-analytics/stream-analytics-define-inputs.md

```diff
@@ -140,6 +148,7 @@ The following table explains each property in the **New input** blade in the Azu
 | **Time format** (optional) |  If you use the time variable in the path, the time format in which the files are organized. Currently the only supported value is `HH`. |
 | **Event serialization format** | The serialization format (JSON, CSV, or Avro) for incoming data streams. |
 | **Encoding** | For CSV and JSON, UTF-8 is currently the only supported encoding format. |
+| **Compression** (optional) | The compression type (None, GZip, or Deflate) of the incoming data stream. |
```

Comment:
Since this compression setting is optional, please call out what is the default value.